pkg/cmd/server/origin/master.go

```diff
@@ -200,7 +202,8 @@ func (c *MasterConfig) RunInProxyMode(proxy *kubernetes.ProxyConfig, assetConfig
 	messages = append(messages, fmt.Sprintf("Started OpenAPI Schema at %%s%s", openAPIServePath))
 
 	// install origin handlers
-	c.InstallProtectedAPI(container)
+	// REBASE: delete proxy mode or make the following work
```

Comment:
@smarterclayton we get to remove it, right?  Never produced a reliable cluster and no the direction we want to proceed in.